File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/ListViews_View.cpp

#include "ListViews_View.h"
#include "ListViews_View.h"
#include <vector>
#include <memory>
#include "ListViews_ViewMyListOfLabelRow.h"
#include "ListViews_ViewMyListOfImagesRow.h"
#include "ListViews_ViewMyCheckboxesRow.h"
#include "ListViews_ViewMyStringRowHandlesRow.h"
#include "ListViews_ViewMyListWithMultiRowSelectionRow.h"
#include "ListViews_ViewMyButtonsRow.h"
#include <string>

namespace widgetassertions
{
  bool ListViews_View::getIsMyListOfLabelListVisible( ) 
  {
    return this->isMyListOfLabelListVisible;
  }
  bool ListViews_View::getIsMyListOfLabelListEnabled( ) 
  {
    return this->isMyListOfLabelListEnabled;
  }
  std::vector<std::shared_ptr<ListViews_ViewMyListOfLabelRow>> ListViews_View::getMyListOfLabelListRows( ) 
  {
    return this->MyListOfLabelListRows;
  }
  int ListViews_View::getMyListOfLabelListSelectedRow( ) 
  {
    return this->MyListOfLabelListSelectedRow;
  }
  bool ListViews_View::getIsMyListOfImagesListVisible( ) 
  {
    return this->isMyListOfImagesListVisible;
  }
  bool ListViews_View::getIsMyListOfImagesListEnabled( ) 
  {
    return this->isMyListOfImagesListEnabled;
  }
  std::vector<std::shared_ptr<ListViews_ViewMyListOfImagesRow>> ListViews_View::getMyListOfImagesListRows( ) 
  {
    return this->MyListOfImagesListRows;
  }
  int ListViews_View::getMyListOfImagesListSelectedRow( ) 
  {
    return this->MyListOfImagesListSelectedRow;
  }
  std::vector<std::shared_ptr<ListViews_ViewMyCheckboxesRow>> ListViews_View::getMyCheckboxesListRows( ) 
  {
    return this->MyCheckboxesListRows;
  }
  int ListViews_View::getMyCheckboxesListSelectedRow( ) 
  {
    return this->MyCheckboxesListSelectedRow;
  }
  bool ListViews_View::getIsMyCheckboxesListVisible( ) 
  {
    return this->isMyCheckboxesListVisible;
  }
  bool ListViews_View::getIsMyCheckboxesListEnabled( ) 
  {
    return this->isMyCheckboxesListEnabled;
  }
  std::vector<std::shared_ptr<ListViews_ViewMyStringRowHandlesRow>> ListViews_View::getMyStringRowHandlesListRows( ) 
  {
    return this->MyStringRowHandlesListRows;
  }
  std::string ListViews_View::getMyStringRowHandlesListSelectedRow( ) 
  {
    return this->MyStringRowHandlesListSelectedRow;
  }
  std::vector<std::shared_ptr<ListViews_ViewMyListWithMultiRowSelectionRow>> ListViews_View::getMyListWithMultiRowSelectionListRows( ) 
  {
    return this->MyListWithMultiRowSelectionListRows;
  }
  std::vector<int> ListViews_View::getMyListWithMultiRowSelectionListSelectedRows( ) 
  {
    return this->MyListWithMultiRowSelectionListSelectedRows;
  }
  std::vector<std::shared_ptr<ListViews_ViewMyButtonsRow>> ListViews_View::getMyButtonsListRows( ) 
  {
    return this->MyButtonsListRows;
  }
  void ListViews_View::buttonsClicked(int rowIndex ) 
  {
    
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/ListViews_View.cs
/// <filename>
///     ListViews_View.cs
/// </filename>
using System.Collections.Generic;
using widgetassertions;

namespace widgetassertions
{
    public class ListViews_View
  {
        private bool isMyListOfLabelListVisible;
     
        public     virtual bool getIsMyListOfLabelListVisible()
    {
      return this.isMyListOfLabelListVisible;
    }
     
        private bool isMyListOfLabelListEnabled;
     
        public     virtual bool getIsMyListOfLabelListEnabled()
    {
      return this.isMyListOfLabelListEnabled;
    }
     
        private System.Collections.Generic.List<widgetassertions.ListViews_ViewMyListOfLabelRow> MyListOfLabelListRows = new System.Collections.Generic.List<widgetassertions.ListViews_ViewMyListOfLabelRow>();
     
        public     virtual System.Collections.Generic.List<widgetassertions.ListViews_ViewMyListOfLabelRow> getMyListOfLabelListRows()
    {
      return this.MyListOfLabelListRows;
    }
     
        private int MyListOfLabelListSelectedRow;
     
        public     virtual int getMyListOfLabelListSelectedRow()
    {
      return this.MyListOfLabelListSelectedRow;
    }
     
        private bool isMyListOfImagesListVisible;
     
        public     virtual bool getIsMyListOfImagesListVisible()
    {
      return this.isMyListOfImagesListVisible;
    }
     
        private bool isMyListOfImagesListEnabled;
     
        public     virtual bool getIsMyListOfImagesListEnabled()
    {
      return this.isMyListOfImagesListEnabled;
    }
     
        private System.Collections.Generic.List<widgetassertions.ListViews_ViewMyListOfImagesRow> MyListOfImagesListRows = new System.Collections.Generic.List<widgetassertions.ListViews_ViewMyListOfImagesRow>();
     
        public     virtual System.Collections.Generic.List<widgetassertions.ListViews_ViewMyListOfImagesRow> getMyListOfImagesListRows()
    {
      return this.MyListOfImagesListRows;
    }
     
        private int MyListOfImagesListSelectedRow;
     
        public     virtual int getMyListOfImagesListSelectedRow()
    {
      return this.MyListOfImagesListSelectedRow;
    }
     
        private System.Collections.Generic.List<widgetassertions.ListViews_ViewMyCheckboxesRow> MyCheckboxesListRows = new System.Collections.Generic.List<widgetassertions.ListViews_ViewMyCheckboxesRow>();
     
        public     virtual System.Collections.Generic.List<widgetassertions.ListViews_ViewMyCheckboxesRow> getMyCheckboxesListRows()
    {
      return this.MyCheckboxesListRows;
    }
     
        private int MyCheckboxesListSelectedRow;
     
        public     virtual int getMyCheckboxesListSelectedRow()
    {
      return this.MyCheckboxesListSelectedRow;
    }
     
        private bool isMyCheckboxesListVisible;
     
        public     virtual bool getIsMyCheckboxesListVisible()
    {
      return this.isMyCheckboxesListVisible;
    }
     
        private bool isMyCheckboxesListEnabled;
     
        public     virtual bool getIsMyCheckboxesListEnabled()
    {
      return this.isMyCheckboxesListEnabled;
    }
     
        private System.Collections.Generic.List<widgetassertions.ListViews_ViewMyStringRowHandlesRow> MyStringRowHandlesListRows = new System.Collections.Generic.List<widgetassertions.ListViews_ViewMyStringRowHandlesRow>();
     
        public     virtual System.Collections.Generic.List<widgetassertions.ListViews_ViewMyStringRowHandlesRow> getMyStringRowHandlesListRows()
    {
      return this.MyStringRowHandlesListRows;
    }
     
        private string MyStringRowHandlesListSelectedRow;
     
        public     virtual string getMyStringRowHandlesListSelectedRow()
    {
      return this.MyStringRowHandlesListSelectedRow;
    }
     
        private System.Collections.Generic.List<widgetassertions.ListViews_ViewMyListWithMultiRowSelectionRow> MyListWithMultiRowSelectionListRows = new System.Collections.Generic.List<widgetassertions.ListViews_ViewMyListWithMultiRowSelectionRow>();
     
        public     virtual System.Collections.Generic.List<widgetassertions.ListViews_ViewMyListWithMultiRowSelectionRow> getMyListWithMultiRowSelectionListRows()
    {
      return this.MyListWithMultiRowSelectionListRows;
    }
     
        private System.Collections.Generic.List<int> MyListWithMultiRowSelectionListSelectedRows = new System.Collections.Generic.List<int>();
     
        public     virtual System.Collections.Generic.List<int> getMyListWithMultiRowSelectionListSelectedRows()
    {
      return this.MyListWithMultiRowSelectionListSelectedRows;
    }
     
        private System.Collections.Generic.List<widgetassertions.ListViews_ViewMyButtonsRow> MyButtonsListRows = new System.Collections.Generic.List<widgetassertions.ListViews_ViewMyButtonsRow>();
     
        public     virtual System.Collections.Generic.List<widgetassertions.ListViews_ViewMyButtonsRow> getMyButtonsListRows()
    {
      return this.MyButtonsListRows;
    }
     
        public     virtual void buttonsClicked(int rowIndex)
    {
      
    }
  }
}

